src/constants/DesignTokens.ts

```diff
@@ -26,6 +26,11 @@ export const borderWidth = {
   medium: 2
 } as const;
 
+export const fonts = {
+  "PlusJakartaSans-Regular": "PlusJakartaSans_400Regular",
+  "PlusJakartaSans-Bold": "PlusJakartaSans_700Bold"
+};
```

Comment:
I would refactor it to use underline instead of dashes, it's better to be accessed when using its object
```suggestion
export const fonts = {
  "PlusJakartaSans_Regular": "PlusJakartaSans_400Regular",
  "PlusJakartaSans_Bold": "PlusJakartaSans_700Bold"
};
```
With that, you could access it like `fonts.PlusJakartaSans_Regular` instead of `fonts["PlusJakartaSans-Regular"]` like you're doing right now.